week3/golf.hs

```diff
@@ -0,0 +1,33 @@
+{-# OPTIONS_GHC -Wall #-}
+
+module Golf where
+
+import Data.List
+  
+-- Exercise 1
+skips :: [a] -> [[a]]
+skips [] = []
+skips a@(_:xs) = a : skips xs
```

Comment:
For `[a,b,c,d]` your `skips` returns: `[[a,b,c,d], [b,c,d], [c,d], [d]]`, but it should return `[[a,b,c,d], [b,d], [c], [d]]`

Reply:
Fixed